test_utils/src/lib.rs

```diff
@@ -476,7 +476,15 @@ pub fn start_holochain_instance<T: Into<String>>(
     let test_config = env::var("INTEGRATION_TEST_CONFIG").map(|test_config|{
             if test_config =="lib3h"
             {
-                TestNodeConfig::MemoryGhostEngine(vec![])
+                let bootstrap_nodes = env::var("BOOTSTRAP_NODES").map(|bootstrap_config|{
+                    
+                    //https://bootstrap1,http://boostrap2 in the env config to set up before tests
+                     bootstrap_config.split(",").map(|s|{
+                         url::Url::parse(s).expect(&format!("Could not parse thus could not set up test for run {:?}",&s))
```

Comment:
`.expect` evaluates its argument regardless of it uses it. To avoid computing this `format!` in the happy path, I suggest using `.unwrap_or_else(|_| panic!(...))` instead. Also might as well include the parse error.
```suggestion
                         url::Url::parse(s).unwrap_or_else(|e| panic!("failed to set up for test run: failed to parse as url: (input: {:?}, error: {:?})",&s, e))
```